### DISCUSSION TITLE
I Cant find a higher version of atoti

### DISCUSSION THREAD

**Discussion by GameTn** [atoti/atoti#566] [Q&A] [answered]
Good Morning , i'm using atoti in other pc and i tried to install atoti `0.6.5` , but it keeps installing the older version , i tried to uninstall and re-install but its didnt work . Is there any way to do it manually ?

**Comment by tibdex** [comment, marked as answer]
Can you please share the command you're running and its output?

**Reply by GameTn** [reply]
```console
$ pip install --upgrade atoti
Requirement already satisfied
```

```console
$ pip install atoti==0.6.5
ERROR: Could not find a version that satisfies the requirement atoti==0.6.5 (from versions: 0.0.1, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.5.0, 0.5.1, 0.
5.2, 0.5.3, 0.5.4, 0.5.5, 0.6.0, 0.6.1, 0.6.2, 0.6.3, 0.6.4)
ERROR: No matching distribution found for atoti==0.6.5
```

**Reply by tibdex** [reply]
atoti 0.6.5 dropped the support of Python 3.7, maybe you're using this version and pip thus not suggests atoti 0.6.5?

**Reply by GameTn** [reply]
The version of python is : `Python 3.9.7`

**Reply by tibdex** [reply]
atoti >= 0.6.5 explicitly rejects Python 3.9.7 as mentioned https://github.com/atoti/atoti/issues/528#issuecomment-1075360882. You can retry with Python > 3.9.7.

**Reply by GameTn** [reply]
Thank you , Problem Solved
